transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/HyperwalletTransferMethodUi.java

```diff
@@ -102,4 +103,9 @@ public Intent getIntentAddTransferMethodActivity(@NonNull final Context context,
         intent.putExtra(EXTRA_TRANSFER_METHOD_PROFILE_TYPE, profileType);
         return intent;
     }
+
+    public static void clearInstance() {
+        sInstance = null;
+        Hyperwallet.clearInstance();
+    }
```

Comment:
Add clearInstance method inside HyperwalletInsight class and add HyperwalletInsight.clearInstance() here.

Reply:
It is merged. May I do in another PR?